recipes-devtools/rust/rust.inc

```diff
@@ -47,6 +47,11 @@ def llvm_features_from_tune(d):
         return []
     feat = frozenset(feat.split())
 
+    mach_overrides = d.getVar('MACHINEOVERRIDES')
+    if not mach_overrides:
```

Comment:
Why not proceed with an empty mach_overrides instead of aborting entirely?

Reply:
@srwalter I've removed the return and rebased this branch to the latest master as well.